app/public/templates/scene.html

```diff
@@ -22,7 +22,8 @@
 
   <body>
     <div id="scene-root">
-      <a-scene cursor="rayOrigin:mouse" raycaster="objects: .link">
+      <!-- Need to add  inspector="url: http://localhost:3333/dist/aframe-inspector.js" to the a-scene element if using local inspector -->
+      <a-scene id="{{ scene_id }}-scene" cursor="rayOrigin:mouse" raycaster="objects: .link">
```

Comment:
We don't need the extra variable here, you should be able to use scene directly aka:
```
{{ scene_dict['id'] }}-scene
```